ci_environment/google-chrome/recipes/default.rb

```diff
@@ -0,0 +1,51 @@
+#
+# Cookbook Name:: google-chrome
+# Recipe:: default
+# Copyright 2015, Travis CI Development Team <contact@travis-ci.org>
+
+chrome_file = File.join(Chef::Config[:file_cache_path], 'chrome.deb')
+
+deps = %w(
+  gconf-service
+  libasound2
+  libcairo2
+  libcups2
+  libfontconfig1
+  libgconf-2-4
+  libgdk-pixbuf2.0-0
+  libgtk2.0-0
+  libnspr4
+  libnss3
+  libpango1.0-0
+  libx11-6
+  libxcomposite1
+  libxcursor1
+  libxdamage1
+  libxext6
+  libxfixes3
+  libxi6
+  libxrandr2
+  libxrender1
+  libxss1
+  libxtst6
+  libdbusmenu-glib4
+  libdbusmenu-gtk4
+  libindicator7
+  libappindicator1
+  libcurl3
+  xdg-utils
+)
+
+bash "install dependencies" do
+  user 'root'
+  code "apt-get install #{deps.join(' ')}"
```

Comment:
I think get why this is being used instead of the `package` resource, but it'd be cool to have a comment describing as much.